Firebase/Core/Public/FIRTimestamp.h

```diff
@@ -19,41 +19,29 @@
 NS_ASSUME_NONNULL_BEGIN
```

Comment:
Can `NS_ASSUME_NONNULL` be used in FirebaseCore?

Reply:
Yes.